<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>importWidget</class>
 <widget class="QWidget" name="importWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>587</width>
    <height>492</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string comment="A books title">Title</string>
       </property>
       <property name="buddy">
        <cstring>titleEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="titleEdit">
       <property name="toolTip">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="authorLabel">
       <property name="text">
        <string comment="A person who writes books.">Author</string>
       </property>
       <property name="buddy">
        <cstring>authorEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="authorEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="relNumLabel">
         <property name="text">
          <string comment="A books edition or release number.">Release Number</string>
         </property>
         <property name="buddy">
          <cstring>relNumEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KLineEdit" name="relNumEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="relDateLabel">
         <property name="text">
          <string>Release Date</string>
         </property>
         <property name="buddy">
          <cstring>relDateEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KLineEdit" name="relDateEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="seriesLabel">
       <property name="text">
        <string>Series</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="seriesEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="volumeLabel">
       <property name="accessibleName">
        <string comment="The volume number of a book that is part of a series of books."/>
       </property>
       <property name="text">
        <string>Volume</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="volumeEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="genreLabel">
       <property name="text">
        <string comment="A books type or style, examples- Science fiction or history">Genre</string>
       </property>
       <property name="buddy">
        <cstring>genreEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="genreEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="locationLabel">
       <property name="text">
        <string comment="The on disk path to a book.">Location</string>
       </property>
       <property name="buddy">
        <cstring>locationUrlRequestor</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KUrlRequester" name="locationUrlRequestor"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="descLabel">
       <property name="text">
        <string comment="A brief summary or description of the book">Description</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KTextEdit" name="descEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
